app/views/transition_landing_page/_countdown.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="Transition-countdown govuk-grid-row">
```

Comment:
The use of the custom `transition-countdown` classes together with the govuk-frontend grid classes seems to result in a kind of conflict, where the custom classes are having to override aspects of the govuk-frontend classes. Why are the grid classes here at all? It looks like the whole thing could be simpler if they were removed.